src/d3a/setup/two_sided_market/default_2a.py

```diff
@@ -28,7 +28,7 @@
 
 def get_setup(config):
 
-    ConstSettings.IAASettings.MARKET_TYPE = 3
+    ConstSettings.IAASettings.MARKET_TYPE = 2
```

Comment:
What? How did this happen in the first place? Weird! Thanks for fixing!

Reply:
Don't know, perhaps somehow I overlooked it.